### PR TITLE
db_plugin cmake simplification

### PR DESCRIPTION
**Pull request by jgiszczak**

### CODE REVIEW

**Comment by mitza-oci** [comment]
I wish I had seen this before working on #658.  Should be roughly the same change.

**Comment by jgiszczak** [comment]
@mitza-oci,
Yes, but this one fails, and yours might actually work.  I was working on this as filler between my usual tasks and haven't had time to devote to figuring out the oddities of the mongocxx installation.  It's doing things on Travis that it did not do on my machine.

If you can complete yours, I will withdraw this one.

File: plugins/db_plugin/CMakeLists.txt [hunk 1]
```diff
@@ -4,43 +4,22 @@ add_library( db_plugin
...
-
-    # NOTE: For this to work, the PKG_CONFIG_PATH variable (man pkg-config) must be set to point to the
-    # 'lib/pkgconfig' subdirectory of the directory used as the argument to CMAKE_INSTALL_PREFIX when
-    # building libmongocxx and libbsoncxx.
-    pkg_search_module(BSONCXX REQUIRED libbsoncxx)
-    pkg_search_module(MONGOCXX REQUIRED libmongocxx)
-
+    message("--   mongocxx found version \"${MONGOCXX_VERSION}\"")
+    message("--   mongocxx include path \"${MONGOCXX_INCLUDE_DIRS}\"")
+    message("--   mongoc libraries \"${MONGOCXX_LIBRARIES}\"")
```
Comment (spicynoodle) [inline plugins/db_plugin/CMakeLists.txt]: tiny typo on line 15
`message("--   mongocxx libraries \"${MONGOCXX_LIBRARIES}\"")`

**Comment by heifner** [comment]
Fixed via #658

**Pull request closed without merging**